pubnub-kotlin/pubnub-kotlin-core-api/src/commonMain/kotlin/com/pubnub/api/models/consumer/objects/member/MemberInclude.kt

```diff
@@ -0,0 +1,19 @@
+package com.pubnub.api.models.consumer.objects.member
+
+abstract class Include(
+    open val includeCustom: Boolean = false,
+    open val includeStatus: Boolean = false,
+    open val includeType: Boolean = false,
+    open val includeTotalCount: Boolean = false,
+)
+
+class MemberInclude(
+    override val includeCustom: Boolean = false,
+    override val includeStatus: Boolean = false,
+    override val includeType: Boolean = false,
+    override val includeTotalCount: Boolean = false,
+    val includeUser: Boolean = false,
+    val includeUserCustom: Boolean = false,
+    val includeUserType: Boolean = false,
+    val includeUserStatus: Boolean = false,
+) : Include()
```

Comment:
do we maybe want to have some set to true by default? I don't know, but there were some in the old version that were default true

Reply:
For old I left true for includeType and includeStatus:
https://github.com/pubnub/kotlin/blob/934ec49cdea16fd524e31b93c5104c4e1ca523ae/pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/PubNubImpl.kt#L1038

https://github.com/pubnub/kotlin/blob/934ec49cdea16fd524e31b93c5104c4e1ca523ae/pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/PubNubImpl.kt#L861

For new I went with false for all. But we can change it.
What do you think?